docs/guides/ingesters-scaling-up-and-down.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Ingesters scaling up and down"
+linkTitle: "Ingesters scaling up and down"
+weight: 10
+slug: ingesters-scaling-up-and-down
+---
+
+This guide explains how to scale up and down ingesters.
+
+_If you're looking how to run ingesters rolling updates, please refer to the [dedicated guide](./ingesters-rolling-updates.md)._
+
+## Scaling up
+
+Adding more ingesters to a Cortex cluster is considered a safe operation. When a new ingester starts, it will register to the [hash ring](../architecture.md#the-hash-ring) and the distributors will reshard received series accordingly.
+
+No special care is required to take when scaling up ingesters.
```

Comment:
I don't think this is true when running without shard-by-all-labels.

Each time you add an ingester some series move to the new ingester.
Suppose you have replication-factor=3, then a query will consult 3 ingesters who _now_ own the series and take the first 2 answers.
If you have added 2 or more ingesters recently then they will only have very recent data.  Medium-age data is still being uploaded to back-end store by the ingesters who previously owned the series. There will be a gap in the query result.

Reply:
https://github.com/cortexproject/cortex/pull/3893